low-level/files/opt/seagate/sspl/setup/resource_map/lr2/resource_map.py

```diff
@@ -95,3 +79,24 @@ def set_health_data(health_data: dict, status, description=None,
             "recommendation": recommendation,
             "specifics": specifics
         })
+
+
+class CortxManifest(metaclass=ABCMeta):
+    """Abstract class for all other manifest types."""
+
+    name = "manifest"
+
+    def __init__(self):
+        """Initialize manifest."""
+        logging_level = Conf.get(SSPL_CONF,
+            f"{SYSTEM_INFORMATION}>{LOG_LEVEL}", "INFO")
+        init_logging('node-manifest', logging_level)
+
+    @abstractmethod
+    def get_manifest_info(self, rpath):
```

Comment:
Class name is Manifest. Interface can exclude calling it again, in get_manifest_info
Manifest.get_manifest_info(rpath) 

get_data sounds more intuitive. 
Manifest.get_data(rpath)